Header header
string winner_id
geometry_msgs/Point goal_point
